### PR TITLE
[SMALLFIX] [branch-1.6] Workaround a corner case when GCS storage owner is null

### PR DESCRIPTION
**Pull request by ifcharming**

### CODE REVIEW

File: underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java [hunk 1]
```diff
@@ -97,15 +98,21 @@ public static GCSUnderFileSystem createInstance(
...
     GoogleStorageService googleStorageService = new GoogleStorageService(googleCredentials);
 
-    String accountOwnerId = googleStorageService.getAccountOwner().getId();
-    // Gets the owner from user-defined static mapping from GCS account id to Alluxio user name.
-    String owner = CommonUtils.getValueFromStaticMapping(
-        conf.getValue(PropertyKey.UNDERFS_GCS_OWNER_ID_TO_USERNAME_MAPPING), accountOwnerId);
-    // If there is no user-defined mapping, use the display name.
-    if (owner == null) {
-      owner = googleStorageService.getAccountOwner().getDisplayName();
+    // getAccountOwner() can return null
```
Comment (madanadit) [inline underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: when is null a valid return value for an authenticated user?

Comment (ifcharming) [reply underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: in my test setup, I'm authenticated. but jets3t can not return a not-null account owner.... I'm not sure why this is the case, maybe something changed on Google Cloud service side... I can't find the root cause yet. So I can only treat this as a possible corner case.

Comment (madanadit) [reply underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: can you leave a TODO?

Comment (ifcharming) [reply underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: updated the comment and added a TODO. thanks

File: underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java [hunk 2]
```diff
@@ -97,15 +98,23 @@ public static GCSUnderFileSystem createInstance(
...
     GoogleStorageService googleStorageService = new GoogleStorageService(googleCredentials);
 
-    String accountOwnerId = googleStorageService.getAccountOwner().getId();
-    // Gets the owner from user-defined static mapping from GCS account id to Alluxio user name.
-    String owner = CommonUtils.getValueFromStaticMapping(
-        conf.getValue(PropertyKey.UNDERFS_GCS_OWNER_ID_TO_USERNAME_MAPPING), accountOwnerId);
-    // If there is no user-defined mapping, use the display name.
-    if (owner == null) {
-      owner = googleStorageService.getAccountOwner().getDisplayName();
+    // getAccountOwner() can return null even when the account is authentication.
```
Comment (madanadit) [inline underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: *authenticated

File: underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java [hunk 3]
```diff
@@ -97,15 +98,23 @@ public static GCSUnderFileSystem createInstance(
...
+    // TODO(chaomin): investigate the root cause why Google cloud service is returning
+    // null StorageOwner.
+    StorageOwner storageOwner = googleStorageService.getAccountOwner();
+    String accountOwnerId = "unknown";
+    String accountOwner = "unknown";
+    if (storageOwner != null) {
```
Comment (madanadit) [inline underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: LOG.warn if null?

File: underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java (hunk 2 above)
Comment (ifcharming) [reply underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: ooops, fixed.

File: underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java (hunk 3 above)
Comment (ifcharming) [reply underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: done, thanks!

**Pull request merged**